sjb/config/test_cases/test_branch_origin_end_to_end.yml

```diff
@@ -30,11 +32,20 @@ extensions:
           make build-images
           docker tag openshift/origin-metrics-server:latest "openshift/origin-metrics-server:$( cat ./ORIGIN_COMMIT )"
         fi
+    - type: "script"
+      title: "build the web console container image"
+      repository: "origin-web-console-server"
+      timeout: 3600
+      script: |-
+        if [[ "${PULL_BASE_REF}" == "master" ]]; then
```

Comment:
or release-3.9?
https://github.com/openshift/aos-cd-jobs/blob/master/sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml#L81

Reply:
I was hesitant to add that since origin-web-console-server does not have a release-3.9 branch (which prompted the question above)